pika/exceptions.py

```diff
@@ -169,3 +169,9 @@ class InvalidMaximumFrameSize(ProtocolSyntaxError):
 
     def __repr__(self):
         return 'AMQP Maximum Frame Size is 131072 Bytes'
+
+
+class ShortStringTooLong(AMQPError):
+
+    def __repr__(self):
+        return 'AMQP Short String can contain up to 255 bytes'
```

Comment:
To enhance debugging, I think it would be good to print the first few hundred bytes of the value at `args[0]`, e.g.,

`return 'AMQP Short String can contain up to 255 bytes: %.300s' % self.args[0]` or
`return 'AMQP Short String can contain up to 255 bytes: %.300s' % self.args`

(working on my first code review from 35000 feet, compliments of JetBlue Beta WiFly :) )